app/flatpak-builtins-build.c

```diff
@@ -452,7 +452,7 @@ flatpak_builtin_build (int argc, char **argv, GCancellable *cancellable, GError
   /* Never set up an a11y bus for builds */
   run_flags |= FLATPAK_RUN_FLAG_NO_A11Y_BUS_PROXY;
 
-  if (!flatpak_run_setup_base_argv (bwrap, runtime_files, app_id_dir, runtime_ref_parts[1], runtime_ref_parts[2],
+  if (!flatpak_run_setup_base_argv (bwrap, runtime_files, app_id_dir, runtime_ref_parts[1], runtime_ref_parts[2], metakey,
```

Comment:
Surely this should be `id`, not `runtime_ref_parts[1]`? We want the app id, not the runtime id?
But there is also the general question of what do we put into the build vs run, because we don't want host settings to contaminate a build, they should normally be isolated (except possibly when using flatpak build to test the app).